app/javascript/readingList/readingList.jsx

```diff
@@ -60,8 +61,24 @@ export class ReadingList extends Component {
         filters: `status:${statusView}`,
       },
     });
+    this.getCollections(currentUser.id)
   }
 
+  getCollections = (user_id) => {
+    const url =`api/${user_id}/collections`
+    console.log('URL!!!! ->', url)
+  fetch(url)
+  .then(response => {
+    return response.json()})
+  .then((data) => {
+    console.log('DATA!!! ->',data)
```

Comment:
out of curiousity, are the `console.log` lines simply used to help debug when you were building this or are they needed when making fetch requests? Asking for my own understanding

Reply:
just working on the prototype.  They will be removed and are not needed for the fetch request.  We are attempting to see what data we would be getting.

Reply:
Cool, got it. Just wanted to have it straight in my head what was going on, thanks for clarifying